charts/cluster/test/monitoring/01-monitoring_cluster.yaml

```diff
@@ -18,8 +18,9 @@ cluster:
               description: "Cache hit ratio"
 backups:
   enabled: false
-pooler:
-  enabled: true
-  instances: 1
-  monitoring:
-    enabled: true
+poolers:
+  - name: rw
+    type: rw
+    instances: 1
+    monitoring:
+      enabled: true
```

Comment:
Add at least two poolers to the test to verify the code above generates the YAML correctly.

Reply:
I added 2 poolers into poolers test, was expect it was enough, but sure, can add here too.

Reply:
Oh! I didn't realize that. But this is also good. Thanks.